packages/compiler/babel.ts

```diff
@@ -0,0 +1,64 @@
+import * as t from '@babel/types';
+import type * as babel from '@babel/core';
```

Comment:
Just a note, we can create an swc plugin for this in the future.

Reply:
yes, for sure. next optimization for the compiler should be flattening blocks, not just props diff gen